### DISCUSSION TITLE
Alertmanager API integration with Sendgrid

### DISCUSSION THREAD

**Discussion by dnaranjor** [prometheus-operator/kube-prometheus#1655] [Q&A]
Does anyone here knows if it exists a way to integrate Alertmanager with the Sendgrid WEB API? 

Currently I'm using the SMTP API integration, but I'd like to take advantage of the WEB API instead

**Comment by metalmatze** [comment]
Yes, it should be possible as I've done that in the past. It needs to be added to the alertmanager configuration.
Generally speaking, this is more of a alertmanager support question than a kube-prometheus one.
